xml/assets/upgrade-2.10-htmldiff.xsl

```diff
@@ -77,6 +77,9 @@
         .count,.data { font-family: monospace;
                        background-color: #F8F8FF;
                        border: 1px solid #DCDCDC; }
+        .err_warning { color: red; background-color: #FFE4E1; }
+        .err_info { color: green; background-color: #FAFAD2; }
```

Comment:
Not required to change, but a comment on accessibility: despite the widespread usage of red and green to indicate meaning, this is the sole color combination that is indistinguishable to most color-blind people (blue-yellow color blindness being the second most common).

Reply:
That's what I call thoughtful review :-)

Hopefully addons (compositor-level filters?) exist to make life of such people
easier; yes, I am washing my hands of this, picking the straightforward named
colours was a way to avoid spending way too much time on a visual aspect of
something that's only useful for a very narrow group of people anyway
(hence the Spartan look, overall) + it's not that important (labels are still displayed)
and perhaps the background colours can still discern the importance.